app/models/miq_action.rb

```diff
@@ -848,6 +848,44 @@ def action_container_image_analyze(action, rec, inputs)
     rec.scan
   end
 
+  def action_container_image_annotate_deny_execution(action, rec, inputs)
+    unless rec.kind_of?(ContainerImage)
+      MiqPolicy.logger.error("MIQ(#{__method__}): Unable to perform action [#{action.description}],"\
+                             " object [#{rec.inspect}] is not a Container Image")
```

Comment:
I'm not sure why `\` is here?  3 of these error messages all have the same prefix:
`MIQ(#{__method__}): Unable to perform action [#{action.description}]`

Can you put that in an error_prefix variable so these logger lines are no so long and therefore remove the need for `\`?